recipes/llm-voice-assistant/ios/PicoLLMVoiceAssistantDemo/ViewModel.swift

```diff
@@ -0,0 +1,335 @@
+//
+//  Copyright 2024 Picovoice Inc.
+//  You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+//  file accompanying this source.
+//  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+//  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+//  specific language governing permissions and limitations under the License.
+//
+
+import Porcupine
+import Cheetah
+import PicoLLM
+import Orca
+import ios_voice_processor
+
+import Combine
+
+enum ChatState {
+    case WAKEWORD
+    case STT
+    case GENERATE
+    case ERROR
+}
+
+class ViewModel: ObservableObject {
+
+    private let ACCESS_KEY = "${YOUR_ACCESS_KEY_HERE}"
+
+    private var porcupine: Porcupine?
+    private var cheetah: Cheetah?
+    private var orca: Orca?
+
+    private var picollm: PicoLLM?
+    private var dialog: PicoLLMDialog?
+
+    private var chatState: ChatState = .WAKEWORD
+
+    private var audioStream: AudioPlayerStream?
+
+    static let modelLoadStatusTextDefault = """
+Start by loading a `.pllm` model file.
+
+You can download directly to your device or airdrop from a Mac.
+"""
+    @Published var modelLoadStatusText = modelLoadStatusTextDefault
+    @Published var enableLoadModelButton = true
+    @Published var showFileImporter = false
+    @Published var selectedModelUrl: URL?
+
+    @Published var enginesLoaded = false
+
+    static let statusTextDefault = "Say `Picovoice` to start!"
```

Comment:
Since we show this every time before transcription/prompting, it makes more sense to just omit the 'to start'
```suggestion
    static let statusTextDefault = "Say `Picovoice`!"
```